chore(docs): Adapters (#38233)
gatsbyjs/gatsby: docs/docs/how-to/previews-deploys-hosting/headers.md

      },
    ],
  },
  {
    source: `/some-path/*`,
    headers: [
      {
        key: `x-custom-header`,
        value: `lose-2`,
      },
    ],
  },
  {
    source: `/some-path/:slug`,
    headers: [
      {
        key: `x-custom-header`,
        value: `win`,
      },
    ],
  },
]

// => Match for path "/some-path/foo"

const output = [{ key: `x-custom-header`, value: `win` }]
```

## Path matching

As you've already seen in the previous paragraphs, you can not only define static `source` paths like `/some-path/` but also ones with path matching. This allows you to target more than one path, opening up more flexibility. You can currently use two path matchers:

- Dynamic: Matching a path segment (no nested paths)
- Wildcard: Matching every path segment after its definition

### Dynamic path

You can use `:` to declare a dynamic path, making `/some-path/:slug` match e.g. `/some-path/foo` and `/some-path/bar`.

```js:title=gatsby-config.js
module.exports = {
  headers: [
    {
      source: `/some-path/:slug`,
      headers: [
        {
          key: `x-custom-header`,
          value: `Hello World`,
        }
      ]
    }
  ]
}
```

### Wildcard path

> You can use `*` to declare a wildcard path, making `/some-path/*` match e.g. `/some-path/foo` and `/some-path/foo/bar`.

Reviewer: Similar edit as above

```suggestion
You can use an asterisk (`*`) to declare a wildcard path. For example, `/some-path/*` will match `/some-path/foo` and `/some-path/foo/bar`.
```
